tests/test_collections.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+import unittest
+from jnius import autoclass, protocol_map
+
+
+class TestCollections(unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCollections, self).__init__(*args, **kwargs)
```

Comment:
just curious why this pass-through override is needed? leftover from tests?

Reply:
yes. i developed the protocol_map in the test constructor before copying into reflect.py. do you want removed?

Reply:
that would be appreciated, not a big deal, but good dead code is deleted code :P.